test/example.md

```diff
@@ -64,6 +67,60 @@ methodName(4, '5', {},
 ```
 
 
+#### ExampleClass()
+
+
+ExampleClass description
+
+
+#### ExampleClass.method2(num)
+
+  - `num: `[`<number>`]
+
+method2 description
+
+
+#### ExampleClass.prototype.constructor(arg1, arg2)
+
+  - `arg1: `[`<Object>`]
+  - `arg2: `[`<string>`]
+
+ExampleClass constructor description
+
+
+#### ExampleClass.prototype.method1(arr)
+
+  - `arr: `[`<Array>`]
+
+method1 description
+
+
+#### PrototypeClass()
+
+
+PrototypeClass description and PrototypeClass
+
+constructor description
+
+
+#### PrototypeClass.method2(num)
+
+  - `num: `[`<number>`]
+
+method1 description
+
+
+#### PrototypeClass.prototype.constructor()
```

Comment:
Will this always be as an empty (no description) method?

Reply:
I can avoid generating documentation for functions with empty signatures, though I don't think we should do that because as for me having function without description is better than not having a function at all. Or we can make a special case for constructors. WDYT?

Reply:
If we find a way to separate the constructor description from the class description here https://github.com/metarhia/metadoc/pull/4#discussion_r244987702 then we should just put that part here.
Otherwise, I think it will be fine to either copy the class description here or just leave it empty.

I don't think we should remove empty methods.

Reply:
If constructor signature is empty then it will be copied from class signature.